jetty-slf4j-impl/src/main/java/org/eclipse/jetty/logging/StdErrAppender.java

```diff
@@ -0,0 +1,331 @@
+        this.strictFormat = config.getBoolean(STRICT_SLF4J_FORMAT_KEY, true);
+        this.threadPadding = config.getInt(THREAD_PADDING_KEY, -1);
+    }
+
+    @Override
+    public void emit(JettyLogger logger, Level level, long timestamp, String threadName, String message)
+    {
+        emit(logger, level, timestamp, threadName, null, message, EMPTY_ARGS);
+    }
+
+    @Override
+    public void emit(JettyLogger logger, Level level, long timestamp, String threadName, Throwable throwable, String message)
+    {
+        emit(logger, level, timestamp, threadName, throwable, message, EMPTY_ARGS);
+    }
+
+    @Override
+    public void emit(JettyLogger logger, Level level, long timestamp, String threadName, String message, Object... argumentArray)
+    {
+        Throwable cause = NormalizedParameters.getThrowableCandidate(argumentArray);
+        emit(logger, level, timestamp, threadName, cause, message, argumentArray);
+    }
+
+    @Override
+    public void emit(JettyLogger logger, Level level, long timestamp, String threadName, Throwable throwable, String message, Object... argumentArray)
+    {
+        StringBuilder builder = new StringBuilder(64);
+        format(builder, logger, level, timestamp, threadName, throwable, message, argumentArray);
+        stderr.println(builder);
+    }
+
+    public ZoneId getTimeZone()
+    {
+        return timezone;
+    }
+
+    public boolean isCondensedNames()
+    {
+        return condensedNames;
+    }
+
+    public boolean isEscapedMessages()
+    {
+        return escapedMessages;
+    }
+
+    public int getThreadPadding()
+    {
+        return threadPadding;
+    }
+
+    public boolean isStrictFormat()
+    {
+        return strictFormat;
+    }
+
+    public void setStream(PrintStream stream)
+    {
+        this.stderr = stream;
+    }
+
+    private void format(StringBuilder builder, JettyLogger logger, Level level, long timestamp, String threadName, Throwable throwable, String message, Object... argumentArray)
+    {
+        Throwable cause = throwable;
+
+        // Timestamp
+        ZonedDateTime tsInstant = Instant.ofEpochMilli(timestamp).atZone(timezone);
+        timestampFormatter.formatTo(tsInstant, builder);
+
+        // Level
+        builder.append(':').append(renderedLevel(level));
+
+        // Logger Name
+        builder.append(':');
+        if (condensedNames)
+        {
+            builder.append(logger.getCondensedName());
+        }
+        else
+        {
+            builder.append(logger.getName());
+        }
+
+        // Thread Name
+        builder.append(':');
+        builder.append(threadName); // TODO: support TAG_PAD configuration
+        builder.append(':');
+
+        // Message
+        builder.append(' ');
+
+        if (strictFormat)
+        {
+            FormattingTuple ft = MessageFormatter.arrayFormat(message, argumentArray);
+            appendEscaped(builder, ft.getMessage());
```

Comment:
what is this?  Looks like slf4j implementation to me.... and a mode of operation.   We don't want either.
We just want simple formatting as we currently do, if anything different is required then we replace the logging jar.
Modes of operation are just slow and confusing.

So we definitely have to remove the modes.

Are the slf4j helper classes in the API jar?   What are the differences between their format and ours?

Reply:
Our formatter is historical and frankly broken.
I'll remove our formatter (the MessageFormatter is in the slf4j-api jar and handles formatting along with final Throwable logic)

Reply:
OK - but can you explain the differences?

Reply:
MessageFormatter will ...

* perform the `{}` formatting
* handle escaped `\\{}` entries in the input format string
* honor the provided `{}` and not add extra ones that are missing
* handle null argument array entries properly (by not showing `null` in the output)
* skip formatting if arguments array is empty 
* if final argument is a throwable make note of it and return it

Our formatter will ...

* perform the `{}` formatting
* not honor escaped `\\{}` entries
* add missing `{}` entries to process entire arguments array
* format even if arguments array is empty or contains null characters
* format Throwable via `Throwable.toString()` as any other argument in the array

Reply:
OK sounds near enough... and if we have the helpers on the classpath anyway!

Reply:
The lax formatting that Jetty StdErrLog had is now removed